src/CreatorLayer.cpp

```diff
@@ -121,6 +126,13 @@ static void reorderButtons(Args... args) {
             ->setAxisAlignment(AxisAlignment::Start)
     );
     this->addChild(bottomLeftMenu);
+
+    if (GameManager::sharedState()->getGameVariable("0077"))

```

Comment:
this one causes a visual inconsistency with the vanilla game and causes all of the buttons to be rearranged

Reply:
yea thats why i said idk how to fix it in the original message...